collect_app/src/main/res/values/theme.xml

```diff
@@ -87,6 +87,7 @@
         </item>
         <item name="shapeAppearanceLargeComponent">@style/ShapeAppearance.Material3.LargeComponent
         </item>
+        <item name="elevationOverlayEnabled">true</item>
```

Comment:
I wanted to include this to make it clearer that `elevationOverlay` is being used here as it's very easy to miss.